appinfo/routes.php

```diff
@@ -244,6 +244,11 @@
 			'url' => '/api/settings/provisioning/{id}',
 			'verb' => 'POST'
 		],
+		[
+			'name' => 'settings#provision',
+			'url' => '/api/settings/provisioning/all',
+			'verb' => 'GET'
```

Comment:
this is an action, this should be PUT or POST

Reply:
Done, thanks!